tests/phpunit/testcases/core/themeCompatibility.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * @group core
+ * @group core-theme-compatibility
+ */
+class BP_Tests_Theme_Compatibility_Functions extends BP_UnitTestCase {
+
+	/**
+	 * @group bp_current_theme_supports
+	 */
+	public function test_bp_current_theme_doesnot_support() {
+		$this->assertFalse( bp_current_theme_supports() );
+	}
+
+	/**
+	 * @group bp_current_theme_supports
+	 */
+	public function test_bp_current_theme_does_support_buddypress() {
+		add_theme_support( 'buddypress' );
+		$this->assertTrue( bp_current_theme_supports() );
+		remove_theme_support( 'buddypress' );
```

Comment:
theme support is not removed in `tear_down`?

Reply:
Core, for example, doesn't remove it, I assume because it is already removed in tear_down.

https://github.com/WordPress/wordpress-develop/blob/b6a3b9c7d1ce33cbeca6f95871a26c48141e524b/tests/phpunit/tests/theme/getThemeStarterContent.php#L14